tests/endtoend/blob_functions/put_get_multiple_blobs_as_bytes_return_http_response/function.json

```diff
@@ -9,31 +9,31 @@
     {
       "type": "blob",
       "direction": "in",
-      "name": "input_file_1",
+      "name": "inputfile1",
```

Comment:
I started hitting issues with the use of `_` in the binding name. Removed the `_` for resolving the tests. Doesn't affect the test.